### ISSUE TITLE
problem with scipy: 0.15.0-np19py27_p0 [mkl]    64-bit linux,  python 2.7.9

### ISSUE THREAD

**Issue by grlee77** [ContinuumIO/anaconda-issues#240]
Earlier today, I ran: ```conda update scipy``` and my scipy version was upgraded to 0.15.0.  However it seems there is a problem with the build as I get an error on the following import:

```python
from scipy.linalg import calc_lwork
```
```python
Traceback (most recent call last):

  File "<ipython-input-1-a402fe72160f>", line 1, in <module>
    from scipy.linalg import calc_lwork

ImportError: cannot import name calc_lwork
```
If I downgrade the version via ```conda install scipy=0.14.0```
The import works as expected.

From a quick look at the scipy source, it seems that there is a fortran file calc_lwork.f that should get compiled when the scipy package is built.

**Comment by kyleabeauchamp** [comment]
More problems:

```
In [1]: import scipy
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-4dc45f4c0083> in <module>()
----> 1 import scipy

/home/kyleb/opt/lib/python2.7/site-packages/scipy/__init__.py in <module>()
    112         raise ImportError(msg)
    113 
--> 114     from scipy.version import version as __version__
    115     from scipy.lib._version import NumpyVersion as _NumpyVersion
    116     if _NumpyVersion(__numpy_version__) < '1.5.1':

ImportError: No module named version
```

**Comment by kyleabeauchamp** [comment]
Fails on 1.5:

```
python -c "import scipy"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/kyleb/opt/lib/python2.7/site-packages/scipy/__init__.py", line 114, in <module>
    from scipy.version import version as __version__
ImportError: No module named version
```

Downgrading fixes:
```
conda install scipy=0.14
[...]
python -c "import scipy"
# Works
```

**Comment by asmeurer** [comment]
Try `conda install -f scipy`.

**Comment by kyleabeauchamp** [comment]
Works for me.

**Comment by grlee77** [comment]
`conda install -f scipy` did not fix the import error I was getting.  I was always able to do a basic `import scipy` or `import scipy.linalg` without issue.

However, I found that it does import from the following alternate location:
```from scipy.lib.lapack import calc_lwork```

I am now thinking the conda package is fine and it is just that calc_lwork` was moved in scipy v0.15.  I don't use that function directly, but an older version of statsmodels I was using was trying to import it.  I see that in the current master, statsmodels no longer imports calc_lwork at all within statsmodels.tools.linalg
